src/en/network-spaces.md

```diff
@@ -132,9 +136,24 @@ Juju now creates bridges for containers *only* when Juju knows the spaces an
 application may require, and the container's bridge for that application will
 only connect to the required network interfaces. 
 
+## Using spaces
+
+Once all desired spaces have been added and/or configured they are called upon
+when adding a machine or deploying an application. There are two methods that
+can be used for doing this: via a binding or constraint.
+
+A binding operates at the software level and is a more fine-grained request. It
+associates an application endpoint with a subnet.
```

Comment:
It would probably be handy to give an example of what is meant by 'application endpoint' since it is a source of some confusion.

Reply:
There is a fair amount of work surrounding endpoints. I've filed #2479 for this.